keras_cv/models/object_detection/yolo_v8/yolo_v8_detector.py

```diff
@@ -645,6 +645,18 @@ def get_config(self):
             "prediction_decoder": self._prediction_decoder,
```

Comment:
we should probably also call `keras.saving.serialize_keras_object` on `self.label_encoder` and `self._prediction_decoder`, seems not strictly necessary, but still more consistent

Reply:
Fixed, thanks!